widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css

```diff
@@ -5308,6 +5308,10 @@ td.tab-spacer-wrapper {
   border-right: 1px solid #e0e0e0;
 }
 
+#dojox_grid__View_1 .dojoxGridCell:focus-visible {
```

Comment:
The id you are anchoring this selector on is in all likelihood a dynamically generated identifier, and, as such, it would be best to not use it.

There appear to be safe alternatives. For example: `#licensesGrid .dojoxGridView .dojoxGridCell:focus-visible`